gettsim/transfers/grunds_im_alter.py

```diff
@@ -0,0 +1,323 @@
+from gettsim.piecewise_functions import piecewise_polynomial
+from gettsim.typing import BoolSeries
+from gettsim.typing import FloatSeries
+from gettsim.typing import IntSeries
+
+
+def grunds_im_alter_m_hh(
+    grunds_ia_m_minus_eink_hh: FloatSeries,
+    anz_erwachsene_hh: IntSeries,
+    anz_rentner_hh: IntSeries,
+) -> FloatSeries:
+    """Calculate Grundsicherung im Alter on household level.
+
+    # ToDo: There is no check for Wohngeld included as Wohngeld is
+    currently not implemented for retirees.
+
+    Parameters
+    ----------
+    grunds_ia_m_minus_eink_hh
+        See :func:`grunds_ia_m_minus_eink_hh`.
+    anz_erwachsene_hh
+        See :func:`anz_erwachsene_hh`.
+    anz_rentner_hh
+        See :func:`anz_rentner_hh`.
+    Returns
+    -------
+
+    """
+    out = grunds_ia_m_minus_eink_hh.clip(lower=0)
+    cond = anz_erwachsene_hh != anz_rentner_hh
```

Comment:
This could be an extra function. Knowing that a household consists of pensioners is an interesting variable and may be used for other things as well.